packages/svelte-docgen/src/parser/mod.js

```diff
@@ -20,14 +20,11 @@ import {
 	is_type_reference,
 } from "../shared.js";
 
+const AnonTypeLiteralSymbolName = ts.InternalSymbolName.Type;
```

Comment:
The reason it is assigned to a variable is because it is one big enum and saves the time on lookup?

Reply:
I intended to give it a bit more self-descriptive name and potentially optimize the property lookup, but I’m not sure if this has any effect…